etc/profile-m-z/mpv.profile

```diff
@@ -86,6 +86,6 @@ private-bin env,mpv,python*,waf,youtube-dl,yt-dlp
 private-dev
 
 dbus-user none
-dbus-system none
+dbus-system filter
```

Comment:
Applications very rarely need access to the system bus. How did you determine the newly released mpv 0.36.0 needs this? In the mean time I'll install the mpv package and do some testing. Thanks for opening the PR, we should definately check what's going on.

Reply:
Without it:
```
dbus[60]: arguments to dbus_message_new_method_call() were incorrect, assertion "path != NULL" failed in file dbus-message.c line 1373.
This is normally a bug in some application using the D-Bus library.

  D-Bus not built with -rdynamic so unable to print a backtrace
```

Reply:
Cannot reproduce on my Arch Linux box. There have been a few changes to the mpv.profile after releasing 0.9.72 though. If you're on that version, check [this commit](https://github.com/netblue30/firejail/commit/2c5119d751cc78e52922544a52a0332f59304639) and try changing the seccomp line accordingly. Or try to reproduce on a [firejail-git](https://github.com/netblue30/firejail/wiki/Using-firejail-from-git) install.

Also, please open an `issue` on this problem you're facing. Properly fill in the issue template and provide as much details as you can, including a full log running with the --debug flag. I do realize this is a bit of a pain, but without eyes on your exact setup it's hard to pinpoint what's happening on your side.